content/en/docs/concepts/storage/volumes.md

```diff
@@ -11,120 +11,79 @@ weight: 10
+
+Kubernetes supports many types of volumes. A {{< glossary_tooltip term_id="pod" text="Pod" >}}
+can use any number of volume types simultaneously. The lifetime of a Kubernetes volume
+coincides with the lifetime of its pod. Consequently, a volume outlives any containers
+that run within the pod, and data is preserved across container restarts. When a
+pod ceases to exist, the volume is destroyed.
 
 At its core, a volume is just a directory, possibly with some data in it, which
-is accessible to the Containers in a Pod.  How that directory comes to be, the
+is accessible to the containers in a pod. How that directory comes to be, the
 medium that backs it, and the contents of it are determined by the particular
 volume type used.
 
-To use a volume, a Pod specifies what volumes to provide for the Pod (the
-`.spec.volumes`
-field) and where to mount those into Containers (the
-`.spec.containers[*].volumeMounts`
-field).
-
+To use a volume, specify the volumes to provide for the Pod in `.spec.volumes`
+and declare where to mount those volumes into containers in `.spec.containers[*].volumeMounts`.
 A process in a container sees a filesystem view composed from their Docker
-image and volumes.  The [Docker
-image](https://docs.docker.com/userguide/dockerimages/) is at the root of the
-filesystem hierarchy, and any volumes are mounted at the specified paths within
-the image.  Volumes can not mount onto other volumes or have hard links to
-other volumes.  Each Container in the Pod must independently specify where to
+image and volumes. The [Docker image](https://docs.docker.com/userguide/dockerimages/)
+is at the root of the filesystem hierarchy. Volumes mount at the specified paths within
+the image. Volumes can not mount onto other volumes or have hard links to
+other volumes. Each Container in the Pod's configuration must independently specify where to
 mount each volume.
 
-## Types of Volumes
-
-Kubernetes supports several types of Volumes:
-
-   * [awsElasticBlockStore](#awselasticblockstore)
-   * [azureDisk](#azuredisk)
-   * [azureFile](#azurefile)
-   * [cephfs](#cephfs)
-   * [cinder](#cinder)
-   * [configMap](#configmap)
-   * [csi](#csi)
-   * [downwardAPI](#downwardapi)
-   * [emptyDir](#emptydir)
-   * [fc (fibre channel)](#fc)
-   * [flexVolume](#flexVolume)
-   * [flocker](#flocker)
-   * [gcePersistentDisk](#gcepersistentdisk)
-   * [gitRepo (deprecated)](#gitrepo)
-   * [glusterfs](#glusterfs)
-   * [hostPath](#hostpath)
-   * [iscsi](#iscsi)
-   * [local](#local)
-   * [nfs](#nfs)
-   * [persistentVolumeClaim](#persistentvolumeclaim)
-   * [projected](#projected)
-   * [portworxVolume](#portworxvolume)
-   * [quobyte](#quobyte)
-   * [rbd](#rbd)
-   * [scaleIO](#scaleio)
-   * [secret](#secret)
-   * [storageos](#storageos)
-   * [vsphereVolume](#vspherevolume)
-
-We welcome additional contributions.
+## Types of Volumes {#volume-types}
+
+Kubernetes supports several types of volumes.
```

Comment:
is there another place we can see the TOC? I think it's convenient to have a condensed list of all the plugins up front and be able to skip to the relevant detailed section depending on what you're interested in

Reply:
Yes, the TOC for the page (which gets automatically generated from the headings)
is located in the right hand side of the page. The in-page table of contents should contain
each plugin. Is this change okay?

Reply:
yes thanks for explaining!